src/components/header-card-action-buttons/header-card-action-buttons.tsx

```diff
@@ -12,6 +13,7 @@ import { emptyToken, TokenInterface } from 'src/token/interfaces/token.interface
 import { isDefined } from 'src/utils/is-defined';
 import { openUrl } from 'src/utils/linking.util';
 
+import { setIsOnRampPossibilityAction } from '../../store/settings/settings-actions';
```

Comment:
It would be nice to rename it to `setIsOnRampPossibileAction` or `setOnRampPossibilityAction`

Reply:
we already had setIsPromotionEnabledAction so i just follow the existing code style))

Reply:
renamed